mvc/controllers.js

```diff
@@ -73,11 +75,21 @@ exports.patchVotesController = (req, res, next) => {
     } = req
     const promises = [
         patchVotesModel(body, article_id),
-        checkCategoryExists(article_id),
+        checkArticleExists(article_id),
     ]
     Promise.all(promises)
         .then(([updatedArticle]) => {
             res.status(200).send({updatedArticle})
         })
         .catch(next)
 }
+
+exports.deleteCommentByIdController = (req, res, next) => {
+    const { comment_id } = req.params
+    const promises = [checkCommentExists(comment_id), deleteCommentByIdModel(comment_id)]
```

Comment:
Looking at the way you've set this up, you are already throwing a Promise.reject if the comment doesn't exist? Therefore you shouldn't need to use checkCommentExists here 👍